conformance/lang/expressions/logical-expressions/logical_and.balt

```diff
@@ -0,0 +1,732 @@
+Test-Case: error
+Description: Test static type of expression in logical expression via invalid types using literals.
+Labels: logical-expr, literal, list-constructor-expr
```

Comment:
Label `boolean` is missing

Reply:
It is part of the minimal subset right according to [[1]](https://docs.google.com/spreadsheets/d/136xgEre0SpLj4ZkDZww-3SopSYzWi7jN4QRFv1FzYgQ/edit#gid=1787231069)